mathesar_ui/src/components/cell-fabric/data-types/components/SteppedInputCell.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
   import { createEventDispatcher, onMount } from 'svelte';
-
+  import { Truncate } from '@mathesar-component-library';
```

Comment:
Other components from '@mathesar-component-library' are already imported in this component. Truncate can be added along with them. Refer lines 4 to 8.